pumpkin/Cargo.toml

```diff
@@ -62,3 +62,4 @@ mio = { version = "1.0.2", features = ["os-poll", "net"]}
 uuid.workspace = true
 tokio.workspace = true
 rayon.workspace = true
+itertools = "0.13.0"
```

Comment:
Workspace dependency for itertools?

Reply:
Good catch, I just let my editor add the dependency and I didn't notice it landed there instead of the crate 👍